pydantic/_internal/_decorators.py

```diff
@@ -151,9 +151,9 @@ class ModelValidatorDecoratorInfo:
 ]
 
 ReturnType = TypeVar('ReturnType')
-DecoratedType: TypeAlias = (
-    'Union[classmethod[Any, Any, ReturnType], staticmethod[Any, ReturnType], Callable[..., ReturnType], property]'
-)
+DecoratedType: (
+    TypeAlias
+) = 'Union[classmethod[Any, Any, ReturnType], staticmethod[Any, ReturnType], Callable[..., ReturnType], property]'
```

Comment:
This seems odd to me, looking at ruff docs now to learn more...

Reply:
Aha, this is relevant: https://github.com/astral-sh/ruff/issues/8226

Reply:
@Luca-Blight, how many of these issues are fixed if you pull down the latest version of `ruff`?

Reply:
I'm not sure it will cause issue but it certainly is funny looking. I wonder if they have a fix in for it, it will probably not be atleast until the next version.

Reply:
Nice job mentioning the issue in their thread.

Reply:
Yeah looks like they're working hard to get some fixes across the line. I can pull down the latest version tmrw morning and see what's resolved.

Reply:
Fwiw, I shared this thread with the founding/lead engineer. Perhaps they can expedite it the resolution if not available tomorrow.

Reply:
I think we're likely to fix this shortly. (There's a PR up, just trying to make some decisions around the broader annotation formatting before merging.)

Reply:
For reference https://github.com/astral-sh/ruff/pull/8233

Reply:
@charliermarsh @MichaReiser thanks a ton for your work on this awesome tool. I'll keep an eye on that issue with the fix 😄

Reply:
@charliermarsh I see that issue has been resolved. Do you know when you all play to make another release including that fix?

Reply:
I’m planning to cut a release today :)

Reply:
@Luca-Blight, we'll want to revert changes that resemble the example above

Reply:
Out now: https://github.com/astral-sh/ruff/releases/tag/v0.1.3. I also confirmed locally that this change is no longer applied to Pydantic (though you'll want to revert it here -- like Black, if you use this formatting with the parentheses, we do _retain_ it).

Reply:
Awesome, thanks! We reverted the changes 😄 and it's working as expected now 🌟